pgmpy/tests/test_models/test_MarkovModel.py

```diff
@@ -215,13 +215,16 @@ def test_factor_graph(self):
         self.graph.add_factors(phi1, phi2)
 
         factor_graph = self.graph.to_factor_graph()
+        
```

Comment:
@VSSILPA Remove these empty lines. Also, can you check for pep8 errors in your code? You can use
`pycodestyle --max-line-length=120 <filename>` for this check.